### DISCUSSION TITLE
[Support]: Improve Object Detection - Frigate+?

### DISCUSSION THREAD

**Discussion by broyuken** [blakeblackshear/frigate#20417] [General Support]
### Describe the problem you are having

I have a couple of floodlight cameras that are mounted fairly high up, and as such the angle that it sees things at may not be typical. I believe because of this, my dogs keep getting detected as people. 
<img width="1278" height="619" alt="image" src="https://github.com/user-attachments/assets/8af07f67-668b-475d-a2cc-fd2f617b14a7" />

Is there something I can change to help frigate detect them as dogs better? I don't want to change the minimum score, because that seems to be fairly accurate for actual people however dogs seems to be less reliable. Is there a config that says that this is a higher up view, or would frigate+ help out with this at all? the AI summary seems to know what's going on even if Frigate doesn't.

<img width="1273" height="841" alt="image" src="https://github.com/user-attachments/assets/4a7bb4ac-b1eb-4a0e-87c1-662835d99e07" />

### Version

0.16.0-c2f8de9

### What browser(s) are you using?

Version 141.0.7390.55 (Official Build) (64-bit)

### Frigate config file

```yaml
N/A
```

### Relevant Frigate log output

```shell
N/A
```

### Relevant go2rtc log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Frigate stats

```json
N/A
```

### Install method

Docker Compose

### docker-compose file or Docker CLI command

```yaml
N/A
```

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

Reolink Floodlight Eliote

**Comment by inkeep-ai-assistant** [comment]
---

**Frigate+ would significantly help with your situation.** The issue you're experiencing—dogs being misclassified as people due to unusual camera angles—is precisely what Frigate+ is designed to solve.[(1)](https://docs.frigate.video/configuration/object_detectors)

Frigate+ provides community-trained models built from real security camera footage, which delivers far better accuracy than generic models.[(1)](https://docs.frigate.video/configuration/object_detectors) More importantly, Frigate+ includes a fine-tuning workflow where you can upload images from your specific camera angles and correct false positives.[(1)](https://docs.frigate.video/configuration/object_detectors) This allows you to create scene-specific models adapted to your high-mounted cameras.[(1)](https://docs.frigate.video/configuration/object_detectors)

Since your cameras are mounted high with atypical viewing angles, the default model (trained on more standard perspectives) struggles to distinguish dogs from people in these conditions.[(1)](https://docs.frigate.video/configuration/object_detectors) Frigate+ models are trained on a proprietary dataset sourced from real Frigate installations worldwide, providing accuracy beyond generic COCO-based models.[(1)](https://docs.frigate.video/configuration/object_detectors)

**What Frigate+ offers for your use case:**

- Models trained on millions of real security camera images from various angles and conditions[(1)](https://docs.frigate.video/configuration/object_detectors)
- A fine-tuning workflow to upload your own images and correct misclassifications[(1)](https://docs.frigate.video/configuration/object_detectors)
- Expanded object vocabulary with better differentiation between similar objects[(1)](https://docs.frigate.video/configuration/object_detectors)
- Continuous improvement as more community data is contributed[(1)](https://docs.frigate.video/configuration/object_detectors)

The fact that Frigate's AI summary correctly identifies "two dogs on a wooden deck" suggests the underlying image understanding is there—Frigate+ would bring that level of accuracy to the real-time object detection.

For configuration adjustments without Frigate+, you can set per-object type filters with different thresholds, but this won't solve the fundamental misclassification issue.[(2)](https://docs.frigate.video/configuration/reference) You could lower the `threshold` for dogs specifically or adjust `min_score`, but as you noted, this may create other detection issues.[(2)](https://docs.frigate.video/configuration/reference)

**Frigate+ is your best solution here** since it addresses the root cause: training data that doesn't match your camera's perspective.[(1)](https://docs.frigate.video/configuration/object_detectors)

📚 **Sources:**
1. [Supported Hardware](https://docs.frigate.video/configuration/object_detectors)
2. [Full Reference Config](https://docs.frigate.video/configuration/reference)

---

Have another question? Just tag [@inkeep](https://inkeep.com).
